stories/components/SearchInput.stories.js

```diff
@@ -0,0 +1,14 @@
+import { useState } from "react";
+import SearchInput from "../../components/SearchInput";
+
+export default {
+  component: SearchInput,
+  title: "components/Search Input",
+};
+
+const Template = (args) => {
+  const [value, setValue] = useState("");
+  return <SearchInput {...args} value={value} onChange={setValue} />;
+};
+
+export const Basic = Template.bind({});
```

Comment:
Looks good, but I think the Story for the `input` would be best

Reply:
So, the issue #1989 isn't valid anymore. Should I open another issue for `Input` story and make this PR work that way?

Reply:
Yes #1989 is still required but I don't understand why this PR has a new component, it is better to have PR's focused and I still think this PR is trying to make too many changes (I really believe less is more)

Reply:
The issue was to create a story for the `Search` component. So, the only way to use the same `Search` component was to have it in such a way that it can be imported in the story.
Now, when the new `Input` component is also there, I also think that the `Search` component isn't doing much. So, I am suggesting to have a `autoFocus` prop on the `Input` component.
I am suggesting I can create a new issue because we can just create a `Input` story and have a variation under that with a `Search Input` label which uses a different placeholder and the `autoFocus` is also turned on.
I can put both stories in the same PR because it will add only one line and it will solve two issues.

Reply:
My understanding of the issue was to add a Story for the Search page

Reply:
Sorry, I didn't read the issue properly. I will create another PR for this issue.

Reply:
Ok no problem, sorry for the confusion. Usually we prefer the minimum amount of changes and any further improvements discovered by doing the work can come in another issue and PR. Thank you for understanding 👍